src/cli/vector_enrich_text.rs

```diff
@@ -276,6 +215,99 @@ fn vector_enrich_text(
     .into_pipeline_data())
 }
 
+fn chunks_from_input(
+    input: PipelineData,
+    call: &Call,
+    engine_state: &EngineState,
+    stack: &mut Stack,
+) -> Result<Vec<String>, ShellError> {
+    let span = call.head;
+    let mut chunks: Vec<String> = Vec::new();
+
+    let chunk_len = match call.get_flag::<usize>(engine_state, stack, "chunk")? {
+        Some(l) => l,
+        None => 1024,
+    };
+
+    match input.into_value(span) {
+        Value::List { vals, span: _span } => {
+            for v in vals {
+                let rec = match v.as_record() {
+                    Ok(r) => r,
+                    Err(e) => {
+                        return Err(ShellError::GenericError(
+                            "Could not parse list of files".to_string(),
+                            "".to_string(),
+                            None,
+                            None,
+                            vec![e],
+                        ));
+                    }
+                };
+
+                let index = match rec.0.iter().position(|r: &String| *r == "name") {
```

Comment:
I feel like this should be a name_column or something akin to id_column in the doc commands.